From df02d10031b375b53360da7baafcbecb9ddee369 Mon Sep 17 00:00:00 2001
From: Herbert Xu <herbert@gondor.apana.org.au>
Date: Thu, 27 May 2010 11:36:59 +0800
Subject: [VAR] Document local command behaviour change

The localvar nesting changeset causes local to fail when used
outside functions.  While this behaviour is consistent with other
shells, it is a change in dash's beavhiour.

Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
(cherry picked from commit 85d5473923c546779213a6c1c0663fdfb1dbcfba)

Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 ChangeLog |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index 24b7dc6..d29f0e4 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -13,7 +13,7 @@
 
 2010-05-24  Herbert Xu <herbert@gondor.apana.org.au>
 
-	* Add localvars nesting.
+	* Add localvars nesting, local now fails outside functions.
 
 2010-05-03  Gerrit Pape <pape@smarden.org>
 
-- 
1.7.4.1

